doc/source/how-to/releasing.rst

```diff
@@ -212,39 +212,139 @@ script. If you are operating out of a GitHub CI pipeline, email the PyAnsys Core
+Install this tool by running:
 
-.. code-block:: bash
+.. code-block:: shell
 
-   pip install -U pip build twine
+    python -m pip install twine
 
-   REPOSITORY_URL="https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload"
-   python -m twine upload dist/* \
-     -p ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }} \
-     -u PAT \
-     --repository-url $REPOSITORY_URL
 
+.. note:: 
+
+   Once a library has been made public, there is no need to keep uploading new
+   versions of it to the private PyPI.
+
+Uploading to private PyPI using GitHub actions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The following code allows to publish any Python :ref:`Artifacts` contained in
+the ``dist/`` directory to the private PyPI. It is expected to be included when
+:ref:`Using GitHub actions`:
+
+.. code-block:: yaml
+
+    - name: Publish to private PyPI
+      env:
+        TWINE_USERNAME: PAT
+        TWINE_PASSWORD: ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }}
+        TWINE_REPOSITORY_URL: https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload
+      run: |
+        pip install twine
+        python -m twine upload dist/* \
+
+
+Notice that ``PYANSYS_PYPI_PRIVATE_PAT`` needs to be added as a repository
+secret so its value is available during the execution of previous instructions.
+
+Uploading to private PyPI using the command-line
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Alternatively, instead of command-line tool arguments for Twine, you can use environment variables:
 
-.. code::
+.. tabs::
+
+    .. group-tab:: Windows
+
+        .. tabs::
+
+            .. group-tab:: CMD
+
+                .. code-block:: text
+
+                    set TWINE_USERNAME=<PAT>
+                    set TWINE_PASSWORD=<PYANSYS_PYPI_PRIVATE_PAT>
+                    set TWINE_REPOSITORY_URL='https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload'
+
+            .. group-tab:: PowerShell
+
+                .. code-block:: text
+
+                    $env:TWINE_USERNAME=<PAT>
+                    $env:TWINE_PASSWORD=<PYANSYS_PYPI_PRIVATE_PAT>
+                    $env:TWINE_REPOSITORY_URL='https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload'
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            export TWINE_USERNAME=<PAT>
+            export TWINE_PASSWORD=<PYANSYS_PYPI_PRIVATE_PAT>
+            export TWINE_REPOSITORY_URL="https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload"
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            export TWINE_USERNAME=<PAT>
+            export TWINE_PASSWORD=<PYANSYS_PYPI_PRIVATE_PAT>
+            export TWINE_REPOSITORY_URL="https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload"
 
-   export TWINE_USERNAME=PAT
-   export TWINE_PASSWORD=$PYANSYS_PYPI_PRIVATE_PAT
-   export TWINE_REPOSITORY_URL="https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload"
+
+Finally, execute the following command:
```

Comment:
```suggestion
Finally, run the following command:
```

Reply:
I like to consistently use the same terms. The next section uses "running", which is one reason I changed it to run. Plus, run is simpler than execute. Technically writing is always about presenting things as simply as possible.